main.js

```diff
@@ -33,12 +33,13 @@ function createWindow () {
   })
 }
 
-function checkAllUpdates (installedAddonsDict) {
+function checkAllUpdates (installedAddonsDict, configObj) {
   Object.keys(installedAddonsDict).forEach(function (key) {
     checkUpdate(installedAddonsDict[key]).then(checkedStatus => {
-      installedAddonsDict[key].status = checkedStatus
-      saveToAddonList(configObj, installedAddonsDict)
-      console.log(`\tChecking \t${installedAddonsDict[key].displayName}\t${installedAddonsDict[key].version}\t${installedAddonsDict[key].status}`)
+      if (installedAddonsDict[key].status !== checkedStatus){
```

Comment:
is status the same as dlStatus?

Reply:
no, dlStatus object is used to communicate download %